modules/checkout/templates/commerce-checkout-order-summary.html.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * See commerce-order-receipt.html.twig for examples.
+ *
+ * Available variables:
+ * - order_entity: Order entity
+ * - items: The order items
+ * - totals: Result from commerce_order.order_total_summary service
+ * - checkout_flow_step: The current checkout flow step.
+ */
+#}
+{% block order_items %}
```

Comment:
This template is missing a wrapper div with {{ attributes }}, making it impossible to append IDs and classes.